GAD 7 - Radixsort/tests/radix/FastResult.java

```diff
@@ -0,0 +1,25 @@
+package tests.radix;
+
+import java.util.ArrayList;
```

Comment:
As far as I can tell there should be an import for gad.radix.Result here

Reply:
Fixed